### PR TITLE
Update to variants

### PR DESCRIPTION
**Pull request by RLordan**
Hey @rando2, I added some info on the genetic variants of SARS-CoV-2 and one or two edits. Can you suggest reviewers for this section? I was thinking maybe @marouenbg? I would welcome any and all additions/review of this section. I think it probably needs to be linked better to the previous section for fluidity.  
I also added some details on the clinical consequences of these variants. I hope this is relevant. 

Just on another note, this website has decent graphs of what countries the variants P.1, B.1.315, and B.1.1.7 have been detected. It may be a nice figure. 
https://cov-lineages.org/global_report_B.1.1.7.html

Thanks!

### Checklist

- [x] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/09.evolution.md [hunk 1]
```diff
@@ -113,6 +115,63 @@ While they preserve the pathogenicity of the virus, escape mutations change the
...
+B.1.351 was first detected in October 2020 in South Africa, which was later detected in the EU December 28th 2020 and has now spread to at least 26 countries [@url:https://www.gov.za/speeches/minister-zweli-mkhize-confirms-8-725-more-cases-coronavirus-covid-19-18-dec-2020-0000; @url:https://cov-lineages.org/global_report_B.1.351.html; @url:https://virological.org/t/tracking-the-international-spread-of-sars-cov-2-lineages-b-1-1-7-and-b-1-351-501y-v2/592]. 
+B.1.351 contains several mutations at the RBD including K417N, E484K, and N501Y.
+While the biological significance of these mutations are still under investigation, it does appear that this lineage may be associated with increased transmissibility [@doi:10.1101/2020.12.21.20248640], due to the N501Y mutation [@doi:10.1101/2020.12.14.422555; @url:https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563].
```
Comment (rando2) [inline content/09.evolution.md:136]:
```suggestion
While the biological significance of these mutations are still under investigation, it does appear that this lineage may be associated with increased transmissibility [@doi:10.1101/2020.12.21.20248640] due to the N501Y mutation [@doi:10.1101/2020.12.14.422555; @url:https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563].
```

File: content/09.evolution.md [hunk 2]
```diff
@@ -113,6 +115,63 @@ While they preserve the pathogenicity of the virus, escape mutations change the
...
+
```
Comment (rando2) [inline content/09.evolution.md:137]:
```suggestion
```

File: content/09.evolution.md [hunk 3]
```diff
@@ -113,6 +115,63 @@ While they preserve the pathogenicity of the virus, escape mutations change the
...
+B.1.351 contains several mutations at the RBD including K417N, E484K, and N501Y.
+While the biological significance of these mutations are still under investigation, it does appear that this lineage may be associated with increased transmissibility [@doi:10.1101/2020.12.21.20248640], due to the N501Y mutation [@doi:10.1101/2020.12.14.422555; @url:https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563].
+ 
+The P.1 variant is a branch of the B.1.1.28 lineage that was first detected in Japan in samples obtained from four travellers from Brazil during a screening at a Tokyo airport, January 10th 2021 [@url:https://www.ecdc.europa.eu/sites/default/files/documents/COVID-19-risk-related-to-spread-of-new-SARS-CoV-2-variants-EU-EEA-first-update.pdf].
```
Comment (rando2) [inline content/09.evolution.md:138]:
```suggestion
The P.1 variant is a branch of the B.1.1.28 lineage that was first detected in Japan in samples obtained from four travellers from Brazil during a screening at a Tokyo airport on January 10, 2021 [@url:https://www.ecdc.europa.eu/sites/default/files/documents/COVID-19-risk-related-to-spread-of-new-SARS-CoV-2-variants-EU-EEA-first-update.pdf].
```

File: content/09.evolution.md [hunk 4]
```diff
@@ -113,6 +115,63 @@ While they preserve the pathogenicity of the virus, escape mutations change the
...
+Shortly after it was established that there was a concentration of cases of the P.1 variant in Manaus, in the Amazonas State, Brazil.
```
Comment (rando2) [inline content/09.evolution.md:139]:
```suggestion
Shortly thereafter, it was established that there was a concentration of cases of the P.1 variant in Manaus, in the Amazonas State, Brazil.
```

File: content/09.evolution.md [hunk 5]
```diff
@@ -113,6 +115,63 @@ While they preserve the pathogenicity of the virus, escape mutations change the
...
+In a small subset of cases sequenced in Manaus, 42% of COVID-19 cases were identified as the P.1 variant as early as mid December, but seemed to be absent in genome surveillance testing prior to December  [@url:https://virological.org/t/genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-manaus-preliminary-findings/586].
```
Comment (rando2) [inline content/09.evolution.md:140]:
```suggestion
In a small number of samples (n=31) sequenced in Manaus, 42% were identified as the P.1 variant as early as mid-December, but the variant seemed to be absent in genome surveillance testing prior to December [@url:https://virological.org/t/genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-manaus-preliminary-findings/586].
```

File: content/09.evolution.md [hunk 6]
```diff
@@ -113,6 +115,63 @@ While they preserve the pathogenicity of the virus, escape mutations change the
...
+To date, at least eight countries have detected the P.1 lineage [@url:https://cov-lineages.org/global_report_P.1.html].
+While the majority of P.1 cases detected internationally have been linked to travel originating from Brazil, the UK has also reported evidence of community transmission detected via routine community sequencing [@url:https://cov-lineages.org/global_report_P.1.html; @url:https://www.reuters.com/article/us-health-coronavirus-britain-variants/uk-detects-77-cases-of-south-african-covid-variant-nine-of-brazilian-idUSKBN29T07P].  
+P.1 has eight lineage-specific mutations along with three concerning spike protein mutations in the RBD e.g. K417T, E484K, and N501Y [@doi:10.1101/2020.12.21.20248640].
```
Comment (rando2) [inline content/09.evolution.md:143]:
```suggestion
P.1 has eight lineage-specific mutations along with three concerning spike protein mutations in the RBD, including K417T, E484K, and N501Y [@doi:10.1101/2020.12.21.20248640].
```

File: content/09.evolution.md [hunk 7]
```diff
@@ -113,6 +115,63 @@ While they preserve the pathogenicity of the virus, escape mutations change the
...
+
```
Comment (rando2) [inline content/09.evolution.md:144]:
```suggestion
```

File: content/09.evolution.md [hunk 8]
```diff
@@ -113,6 +115,63 @@ While they preserve the pathogenicity of the virus, escape mutations change the
...
+P.1 has eight lineage-specific mutations along with three concerning spike protein mutations in the RBD e.g. K417T, E484K, and N501Y [@doi:10.1101/2020.12.21.20248640].
+ 
+ 
+There have been multiple different SARS-CoV-2 lineages detected that have mostly been of no more clinical concern than the original devastating lineage originating in Wuhan [@url:https://cov-lineages.org/lineages.html].
+However, the spotlight has been cast on other variants of unknown clinical relevance due to the increase of cases observed that have been associated with B.1.1.7 in particular.  
+Although early in its ascendency, CAL.20C (B.1.429) is a SARS-CoV-2 variant that was detected in California, USA [@doi:10.1101/2021.01.18.21249786].
+It was first detected in July 2020 but was not detected again until October 2020. 
+In December 2020, CAL.20C accounted for ~24% of the total cases in Southern California and ~36% of total cases in the Los Angeles area. 
+CAL.20C has now been detected in New York, Washington DC and some cases have been reported outside the USA in Oceania [@doi:10.1101/2021.01.18.21249786].
```
Comment (rando2) [inline content/09.evolution.md:151]:
```suggestion
CAL.20C has now been detected in New York and Washington, D.C., and some cases have been reported outside the USA in Oceania [@doi:10.1101/2021.01.18.21249786].
```

File: content/09.evolution.md [hunk 9]
```diff
@@ -113,6 +115,63 @@ While they preserve the pathogenicity of the virus, escape mutations change the
...
+This variant is characterized by five lineage-specific mutations (ORF1a: I4205V, ORF1b:D1183Y, S: S13I;W152C;L452R).
+The latter spike mutation, L452R, is found in an area of the RBD known to resist monoclonal antibodies to the spike protein [@doi:10.1016/j.cell.2020.07.012] and it is hypothesized that this mutation may resist polyclonal sera in convalescent patients or in individuals post vaccination [@doi:10.1101/2020.12.31.425021; @doi:10.1101/2021.01.18.21249786].
```
Comment (rando2) [inline content/09.evolution.md:153]:
```suggestion
The latter spike mutation, L452R, is found in an area of the RBD known to resist monoclonal antibodies to the spike protein [@doi:10.1016/j.cell.2020.07.012], and it is hypothesized that this mutation may resist polyclonal sera in convalescent patients or in individuals post-vaccination [@doi:10.1101/2020.12.31.425021; @doi:10.1101/2021.01.18.21249786].
```

File: content/09.evolution.md [hunk 10]
```diff
@@ -113,6 +115,63 @@ While they preserve the pathogenicity of the virus, escape mutations change the
...
+CAL.20C is not a designated VOC, however, further research is required to determine the implications of the mutations encoded in this genetic variant.
```
Comment (rando2) [inline content/09.evolution.md:154]:
```suggestion
CAL.20C is not a designated VOC; however, further research is required to determine the implications of the mutations encoded in this genetic variant.  
```

File: content/09.evolution.md [hunk 11]
```diff
@@ -113,6 +115,63 @@ While they preserve the pathogenicity of the virus, escape mutations change the
...
+CAL.20C has now been detected in New York, Washington DC and some cases have been reported outside the USA in Oceania [@doi:10.1101/2021.01.18.21249786].
+This variant is characterized by five lineage-specific mutations (ORF1a: I4205V, ORF1b:D1183Y, S: S13I;W152C;L452R).
+The latter spike mutation, L452R, is found in an area of the RBD known to resist monoclonal antibodies to the spike protein [@doi:10.1016/j.cell.2020.07.012] and it is hypothesized that this mutation may resist polyclonal sera in convalescent patients or in individuals post vaccination [@doi:10.1101/2020.12.31.425021; @doi:10.1101/2021.01.18.21249786].
+CAL.20C is not a designated VOC, however, further research is required to determine the implications of the mutations encoded in this genetic variant.  
+Another notable variant has recently been discovered in 35 patients in a Bavarian hospital in Germany, however, the sequencing data has not been published to date and it remains to be determined whether this variant is of any concern further [@url:https://www.br.de/nachrichten/bayern/neue-coronavirus-mutation-35-faelle-in-garmisch-partenkirchen,SMQ1V6u].
```
Comment (rando2) [inline content/09.evolution.md:155]:
```suggestion
Another notable variant has recently been discovered in 35 patients in a Bavarian hospital in Germany; however, the sequencing data has not been published to date and it remains to be determined whether this variant is of any further concern [@url:https://www.br.de/nachrichten/bayern/neue-coronavirus-mutation-35-faelle-in-garmisch-partenkirchen,SMQ1V6u].
```

**Review by rando2 — APPROVED** [review]
Hi @RLordan! Here is some general feedback on the text. I haven't read too much about this and didn't dig into the citations here, but it seems consistent with the little I've read! I think we should strongly consider including a phylogenetic tree that shows the relationship between the variants and the mutations -- if you happen to have this information in your head and would want to draw a rough sketch, Nils has been able to make gorgeous visualizations of this type of thing. Otherwise, it's something I can work on once I start digging into this section more!

The other contributors who I would suggest might be interested in checking this text out are @marouenbg and @Sergey-Knyazev!

File: content/09.evolution.md [hunk 12]
```diff
@@ -113,6 +115,63 @@ While they preserve the pathogenicity of the virus, escape mutations change the
 By using tools from natural language processing (NLP), viral escape was modeled as an NLP problem [@doi:10.1126/science.abd7331] where a modification makes a sentence grammatically correct but semantically different.
 Therefore, language models of viruses could predict mutations that change the presentation of the virus to the immune system but preserve its infectivity.
 
+#### Variants of concern and variants under surveillance
+
+Viral replication naturally leads to the occurrence of mutations that lead to genetic variants [@doi:10.1038/s41564-020-0690-4].
+The emergence of new genetic variants of SARS-CoV-2 has attracted significant media attention since the detection of a variant of concern (VOC), a distinct phylogenetic cluster referred to as B.1.1.7/VOC 202012/01.
+While the B.1.1.7 lineage garnered attention in November 2020, two genomes of the lineage were detected as early as September 20th 2020 from routine genomic data in Kent, in the United Kingdom (UK) by the COVID-19 Genomics UK Consortium (COG-UK).
```
[the remark this answers is not in the document]
Comment (RLordan) [reply content/09.evolution.md:122]: Probably, I wasn't sure

File: content/09.evolution.md [hunk 13]
```diff
@@ -113,6 +115,63 @@ While they preserve the pathogenicity of the virus, escape mutations change the
...
+Of the twenty-three mutations than define B.1.1.7 from the original strain isolated in Wuhan (lineage A), fourteen are lineage-specific and three appear to be biologically consequential mutations associated with the spike protein, namely N501Y, P681H, and 69-70del [@url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/947048/Technical_Briefing_VOC_SH_NJL2_SH2.pdf;  @url:https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563].
+The latter is a 6bp deletion leading to the loss of two amino acids with consequences for immune recognition, which may be responsible for increased transmissibility of the B.1.1.7 VOC in conjunction with N501Y due to changes in the RBD that mediates viral entry [@url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/947048/Technical_Briefing_VOC_SH_NJL2_SH2.pdf; @doi:10.1101/2020.12.14.422555].
```
[the remark this answers is not in the document]
Comment (RLordan) [reply content/09.evolution.md:126]: I agree with you @rando2 , nice way of putting it. The one thing on our side is the versatility of the mRNA vaccine design. With regards to above, I guess we can adapt part of this section from pathogenesis in order to explain the potential implications of mutation to the Spike. What do you think?

**Comment by RLordan** [comment]
> Hi @RLordan! Here is some general feedback on the text. It looks good to me! I haven't read too much about this and didn't dig into the citations here, but it seems consistent with the little I've read! I think we should strongly consider including a phylogenetic tree that shows the relationship between the variants and the mutations -- if you happen to have this information in your head and would want to draw a rough sketch, Nils has been able to make gorgeous visualizations of this type of thing. Otherwise, it's something I can work on once I start digging into this section more!
> 
> The other contributors who I would suggest might be interested in checking this text out are @marouenbg and @Sergey-Knyazev!

Hey @rando, a phylogenetic tree would be good. I am terrible for sketching and I would be afraid I get it wrong haha! I will try find a figure from a paper as an example, maybe we can show a simplified version :)

**Comment by RLordan** [comment]
Also, just prior to submission I will update the variants to reflect the situation - e.g. B.1.1.7 is now in 70 countries and will rise. Already looking like it will be the dominant variant in a few months.

**Comment by rando2** [comment]
Yiiikes! OK we can definitely keep an eye on this! If you come across any resource that is tracking this in a systematic way, let me know -- we might be able to code it into the manuscript like we have for a few other things!

**Comment by RLordan** [comment]
> Yiiikes! OK we can definitely keep an eye on this! If you come across any resource that is tracking this in a systematic way, let me know -- we might be able to code it into the manuscript like we have for a few other things!
> 
> I am OK at sketching although definitely no artist! I'll keep an eye on this moving forward with this section.

Check out these folks, I think they are doing just that: https://cov-lineages.org/global_report_B.1.1.7.html

They are keeping track of P.1, B.1.1.7, and B.1.351

**Pull request merged**